.idea/compiler.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove all the .idea/* files. 
git rm --cached .idea/*
and then push again